internal/archive/fsops.go

```diff
@@ -121,23 +149,78 @@ func walk(sourcePath string, baseDir string, archive *zip.Writer, ignore map[str
 			return nil
 		}
 
+		if fileInfoProvider.isSymbolicLink(info) {
+			return addSymbolicLinkToArchive(path, baseDir, symLinkPath, linkedPath, archive, ignore)
+		}
+
 		// Don't add the base folder to the zip
 		if info.IsDir() && filepath.Clean(path) == filepath.Clean(baseDir) {
 			return nil
 		}
 
-		// Path in zip should be with slashes (in all operating systems)
-		pathInZip := filepath.ToSlash(getRelativePath(path, baseDir))
-
-		// Folders must end with "/"
-		if info.IsDir() {
-			pathInZip += "/"
-		}
+		pathInZip := getPathInZip(path, baseDir, symLinkPath, linkedPath, info)
 
 		return addToArchive(path, pathInZip, info, archive)
 	})
 }
 
+func getPathInZip(path string, baseDir, symLinkPath, linkedPath string, info os.FileInfo) string {
+	// Path in zip should be with slashes (in all operating systems)
```

Comment:
This comment belongs to the line that calls `filepath.ToSlash`

Reply:
moved